cmd/kogito/command/util/build_from_file.go

```diff
@@ -84,8 +84,11 @@ func ProduceTGZfile(resource string) (io.Reader, error) {
 		}
 		return nil
 	})
+	if err != nil{
+		log.Errorf("%s Error in walking the file", err)
+	}
 	log.Infof(message.KogitoAppFoundFile, filesFound)
-	return &buf, nil
+	return &buf, err
```

Comment:
err here should be nil, otherwise it can cause false positve. return nil, err on the if you added above instead.

Reply:
Didn't understand the comment, this function returns an error, if a there is no error it'll simply return nil. If there is it'll just return that error.

Reply:
yeah, nevermind, here we check for the err as well.

what about just returning the err and remove the if you added?
the error will be returned to cli caller anyways.

@ricardozanini @radtriste @sutaakar wdyt?

Reply:
Error has to be checked here as well or the golangcli-lint will give error for `errcheck`

Reply:
alright then, let's keep it this way.